src/main/java/com/mcmoddev/mmdbot/utilities/dictionary/DictionaryUtils.java

```diff
@@ -42,6 +43,10 @@
 
     private static final HttpClient CLIENT = HttpClient.newHttpClient();
 
+    public static final DictionaryEntry CURLE_ENTRY = new DictionaryEntry("curle", "kəːle",
+        List.of(new DictionaryDefinition("noun", "A seemingly time travelling computer woman with a technological curse, a time-travelling penny, and no fucks to give whatsoever",
```

Comment:
Time travelling appears twice here. Probably worth changing "time-travelling penny" to "teleporting penny" just for the flow of the joke, even if it's not exactly what sci originally said :P